Sources/StreamChat/Utils/Cached.swift

```diff
@@ -1,48 +0,0 @@
-//
-// Copyright © 2024 Stream.io Inc. All rights reserved.
-//
-
-import Foundation
```

Comment:
> why the size of the SDK is not reduced a bit?

Well, it's hard to say. Checked this locally, same result. Perhaps the upside is less than 0.01 and we cut it out from the report.

> why the checks show performance regression?

It looks like something is still odd with channel scrolling. It started last week and has been flaky since then. It's so weird that it was stable for quite so long and now is mostly red 😕

Reply:
Most of the lines removed are just test related, codewise, there was not many changes, so I think that is why there is not significant reduction in code